project/plugins.sbt

```diff
@@ -4,3 +4,4 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.7.0")
 addSbtPlugin("org.tpolecat" % "tut-plugin" % "0.4.8")
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.0")
+addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "latest.release")
```

Comment:
I removed the thoughtworks plugin, added `autoAPIMappings`, and that fixed the errors about the scala std lib classes. I didn't have to add anything else. We would have do this for the typesafe config libraries.